Disable xdebug in master

Currently, it causes compilation failure.

https://travis-ci.org/travis-ci/php-src-builder/jobs/247141204#L2702

```
[xdebug]: Compiling xdebug in /tmp/php-build/source/xdebug-master
-----------------
|  BUILD ERROR  |
-----------------
Here are the last 10 lines from the log:
-----------------------------------------
In file included from /tmp/php-build/source/master/ext/readline/readline.c:28:0:
/tmp/php-build/source/master/ext/readline/php_readline.h:27:2: warning: #warning Readline module will *NEVER* be thread-safe [-Wcpp]
In file included from main/internal_functions_cli.c:65:0:
/tmp/php-build/source/master/ext/readline/php_readline.h:27:2: warning: #warning Readline module will *NEVER* be thread-safe [-Wcpp]
/tmp/php-build/source/master/sapi/phpdbg/phpdbg_io.c: In function ‘phpdbg_output_pager’:
/tmp/php-build/source/master/sapi/phpdbg/phpdbg_io.c:211:10: warning: ignoring return value of ‘write’, declared with attribute warn_unused_result [-Wunused-result]
PEAR package PHP_Archive not installed: generated phar will require PHP's phar extension be enabled.
/tmp/php-build/source/xdebug-master/xdebug_stack.c: In function ‘xdebug_error_cb’:
/tmp/php-build/source/xdebug-master/xdebug_stack.c:803:26: error: ‘zend_executor_globals’ has no member named ‘valid_symbol_table’
make: *** [xdebug_stack.lo] Error 1
-----------------------------------------
The full Log is available at '/tmp/php-build.master.20170626151557.log'.
[Warn]: Aborting build.
```

See also: https://github.com/travis-ci/travis-ci/issues/7959#issuecomment-311360621

--- a/share/php-build/definitions/7.2.0alpha1
+++ b/share/php-build/definitions/7.2.0alpha1
@@ -1,3 +1,2 @@
 install_package "https://downloads.php.net/~pollita/php-7.2.0alpha1.tar.bz2"
-install_xdebug_master
 enable_builtin_opcache

--- a/share/php-build/definitions/7.2.0alpha2
+++ b/share/php-build/definitions/7.2.0alpha2
@@ -1,3 +1,2 @@
 install_package "https://downloads.php.net/~pollita/php-7.2.0alpha2.tar.bz2"
-install_xdebug_master
 enable_builtin_opcache

--- a/share/php-build/definitions/7.2.0alpha3
+++ b/share/php-build/definitions/7.2.0alpha3
@@ -1,3 +1,2 @@
 install_package "https://downloads.php.net/~remi/php-7.2.0alpha3.tar.bz2"
-install_xdebug_master
 enable_builtin_opcache

--- a/share/php-build/definitions/master
+++ b/share/php-build/definitions/master
@@ -1,3 +1,2 @@
 install_package_from_github master
-install_xdebug_master
 enable_builtin_opcache

